### DISCUSSION TITLE
How to stop LightBulb increasing brightness of black level?

### DISCUSSION THREAD

**Discussion by hyyou** [Tyrrrz/LightBulb#336] [Q&A]
Thank you for the excellent program. Everything is functioning well.

However, as an artist, I am particularly sensitive to the black levels. Before activating LightBulb, the darkest pixels on my screen were truly black.

After enabling LightBulb, those pixels appear slightly brighter. Is there a specific setting to adjust brightness without compromising the absolute blackness of the darkest pixels?

**Comment by Tyrrrz** [comment]
Do you mean that the pixels appear brighter even with LightBulb disabled (but running)?

**Reply by hyyou** [reply]
When I start my computer without LightBulb, the darkest pixel on my screen is completely black. However, after opening LightBulb, the darkest pixel becomes a shade of gray. Even if I disable or close LightBulb, the black color does not return.  (Turn off/disable LightBulb impact bright pixels though.) The only way to restore the original black is to restart my computer.

This suggests that LightBulb is altering a temporary setting within Windows, in my opinion.

**Reply by Tyrrrz** [reply]
It only alters the screen gamma, but maybe your default gamma is not an identity matrix (1/1/1), so that's why disabling LightBulb doesn't restore the same colors you'd expect.
